test/Info.rb

```diff
@@ -60,6 +60,8 @@ def test_attenuate
   end
 
   def test_authenticate
```

Comment:
Are these methods deprecated in IM7?

Reply:
This property has been removed in ImageMagick 7.

Reply:
After some more investigation we realized that we could set the `authenticate` option for IM7 to preserve this property. Will update the PR accordingly.